openmdao/utils/coloring.py

```diff
@@ -516,8 +516,7 @@ def _get_bool_jac(prob, repeats=3, tol=1e-15, orders=5, setup=False, run_model=F
                 fullJ += np.abs(J)
         elapsed = time.time() - start_time
 
-    # normalize the full J by dividing by the max value
-    fullJ /= np.max(fullJ)
+    fullJ /= repeats
```

Comment:
looks like the docstring for the function needs to be updated:
"then dividing by the _max value_, then converting to a boolean array, ..."

Reply:
fixed